### DISCUSSION TITLE
Smooth path data transition

### DISCUSSION THREAD

**Discussion by ClemDLS** [motion-canvas/motion-canvas#1073] [Q&A]
Hey, I've got this cute little character (named Chouchou, say hello !). It is only composed of Paths.

![chouchou1](https://github.com/motion-canvas/motion-canvas/assets/114819876/552dddbf-2447-4da5-be47-5920fc21ff23)

I'm trying to animate it, by changing the data property of each Path. But then Chouchou becomes the evil :

![chouchou2](https://github.com/motion-canvas/motion-canvas/assets/114819876/e4d847a2-136f-46bb-a2e7-fa0e02d79320)

I guess, I should use BezierCurves or Splines to keep it smooth during the animation. But this means a lot of work to convert simple paths datas (extractable from the svg file) to knots' position and start/endHandle properties. Am I missing a way to do it more straightforward ?

**Comment by aarthificial** [comment]
Sadly this is a problem with the current implementation but it will be fixed in https://github.com/motion-canvas/motion-canvas/issues/972
